build.gradle

```diff
@@ -25,8 +25,10 @@ repositories {
 }
 
 dependencies {
-    compile 'org.spongepowered:spongeapi:6.0.0-SNAPSHOT'
-    compile 'com.me4502:ModularFramework:1.5'
+    compile 'org.apache.commons:commons-math3:3.6.1'
+    compile "org.spongepowered:spongeapi:${spongeVersion}-SNAPSHOT"
```

Comment:
Include the SNAPSHOT where you specify spongeVersion as not all API releases you will want to use are snapshots